tree/dataframe/inc/ROOT/RDFActionHelpers.hxx

```diff
@@ -323,6 +324,69 @@ public:
    HIST &PartialUpdate(unsigned int slot) { return *fTo->GetAtSlotRaw(slot); }
 };
 
+class FillTGraphHelper : public ROOT::Detail::RDF::RActionImpl<FillTGraphHelper> {
+public:
+   using TGraph = ::TGraph;
+   using Result_t = TGraph;
+
+private:
+   std::shared_ptr<TGraph> fResultGraph;
+   std::unique_ptr<ROOT::TThreadedObject<TGraph>> fTo;
+   bool fIsSortingRequired;
```

Comment:
Since we decided that this is up to the user to trigger, wouldn't this be absent from the helper?